h2o-py/tests/testdir_algos/glm/pyunit_PUBDEV_8421_check_ymu_rank.py

```diff
@@ -0,0 +1,30 @@
+import sys
+sys.path.insert(1,"../../../")
+import h2o
+from tests import pyunit_utils
+from h2o.estimators.glm import H2OGeneralizedLinearEstimator as glm
+
+# check parameters ymu and rank were passed 
+def test_gaussian_alpha():
```

Comment:
Thanks for adding a test for a new functionality, but I'm not sure whether this functionality is worth of the time spend on CI resources (training a new model because in order to check propagation of two fields).

Reply:
Let me make it a very short test.